multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/util/UriUtil.java

```diff
@@ -46,4 +48,14 @@ public static String prettyPrintRoutes(Set<CloudRouteSummary> routes) {
                      .map(CloudRouteSummary::toUriString)
                      .collect(Collectors.joining(", "));
     }
+
+    public static boolean isValidUrl(String url) {
```

Comment:
Wouldn't it be better to call the method `validateUrl` and throw an exception if the URl is invalid, so that the error message shown to the user contains some additional details?